services/consul/Dockerfile

```diff
@@ -21,9 +21,3 @@ rm consul_${CONSUL_VERSION}_web_ui.zip
 # Expose standard ports, as well as ports used by consul
 EXPOSE 8300 8301 8301/udp 8302 8302/udp 8400 8500 53/udp
 
-ADD ./config /consul/config/
-ADD startconsulserver.sh .
-ADD startconsulclient.sh .
-
```

Comment:
we may even want to switch to the public consul image at some point. https://hub.docker.com/_/consul/
no need to maintain our own consul image.